websites/app/src/utils/chains.ts

```diff
@@ -1,149 +1,93 @@
-export const references = [
+export const chains = [
```

Comment:
netlify build errored because there are files that still import `references` like `src/pages/Home/SubmitEntries/AddEntryModal/AddCDN.tsx`